src/main/java/net/datafaker/providers/base/ProviderRegistration.java

```diff
@@ -150,6 +151,8 @@ <PR extends ProviderRegistration, AP extends AbstractProvider<PR>> AP getProvide
 
     RandomService random();
 
+    WeightedRandomSelector weightedRandomSelector();
```

Comment:
what is the reason to have it here?
I would rather keep it completely separate

Reply:
as proposed, it was removed from `ProviderRegistration.java`